### PR TITLE
A Brief Case For Briefcases (the funny exploit fix)

### PR DESCRIPTION
**Pull request by alsoandanswer**
yeah this should hopefully fix it

i have no idea how gear spawning works

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

**Comment by UponASeaOfStars** [comment]
the briefcase bandit has been brought to bear!

**Comment by alsoandanswer** [comment]
> Undocumented change: Makes all the non-secure briefcases 2 points as well. Currently secure is 2 points, others in the selection 1.
> 
> Which is lame. Please no merge of secure and non-secure briefcases.

I wish I could, but it turns out that if you select both in the loadout it'll force one into your backpack anyway (because you already have a briefcase in your hand)

**Comment by UponASeaOfStars** [comment]
This bug has ravaged our community and destroyed countless lives, and I know of at least one nefarious ne'er-do-well who even used it to powergame by putting things in the briefcase! The scourge of Aurora has been removed, and for that, you have my eternal gratitude. May you be blessed as I am blessed to have seen this PR. Godspeedmerge to you, good coder.

in all seriousness though this is a good pr i don't want anyone else getting clowned on

**Review by Arrow768 — DISMISSED** [review]

File: html/changelogs/wezzy_la_exploit.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Fixes recursive loadout storage items."
```
Comment (DreamySkrell) [inline html/changelogs/wezzy_la_exploit.yml:41]: Undocumented change of all briefcases now costing 2 points. Previously they were 1 points, with only the secure briefcase being 2 points.

**Comment by UponASeaOfStars** [comment]
> > Undocumented change: Makes all the non-secure briefcases 2 points as well. Currently secure is 2 points, others in the selection 1.
> > Which is lame. Please no merge of secure and non-secure briefcases.
> 
> I wish I could, but it turns out that if you select both in the loadout it'll force one into your backpack anyway (because you already have a briefcase in your hand)

Regarding this-- would it be possible to have a secure briefcase spawn in your _other_ hand? Left hand for regular, right hand for secure? Or are there already job items that spawn in the hand, which would then cause the briefcase to go into the backpack?

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
document the briefcase point increase in the changelog (and preferably PR description too.)
document the secure briefcase being merged into the briefcase selection.

**Comment by SleepyGemmy** [comment]
this is not really a fix.
a better solution would be to try and spawn huge items in hands first and if not possible, spawn them on the ground.

**Comment by SleepyGemmy** [comment]
not a bugfix PR as it changes the briefcase selection cost.

**Comment by SleepyGemmy** [comment]
this is not really a fix.
a better solution would be to try and spawn huge items in hands first and if not possible, spawn them on the ground, i.e. by checking if the hands are full and if they are, spawning the items on the ground.

**Comment by alsoandanswer** [comment]
> > > Undocumented change: Makes all the non-secure briefcases 2 points as well. Currently secure is 2 points, others in the selection 1.
> > > Which is lame. Please no merge of secure and non-secure briefcases.
> > 
> > 
> > I wish I could, but it turns out that if you select both in the loadout it'll force one into your backpack anyway (because you already have a briefcase in your hand)
> 
> Regarding this-- would it be possible to have a secure briefcase spawn in your _other_ hand? Left hand for regular, right hand for secure? Or are there already job items that spawn in the hand, which would then cause the briefcase to go into the backpack?

Yes, that's the purse

**Comment by alsoandanswer** [comment]
> this is not really a fix. a better solution would be to try and spawn huge items in hands first and if not possible, spawn them on the ground, i.e. by checking if the hands are full and if they are, spawning the items on the ground.

if someone is able and willing to implement this feature, then they are free to revert the changes in this PR

but at the moment, in my opinion, a slightly more expensive loadout item is better than a exploit left in a game.

**Pull request closed without merging**